CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+# Contributing
+
+Pull requests are welcome!
+
+## Getting started
+
+First [fork this repo](https://help.github.com/articles/fork-a-repo/), and
+then clone it & set the upstream repo:
```

Comment:
You terminate this instruction with a ":", but you don't have this with the later instructions. I prefer the ":", but whichever you choose, we should be consistent.